platforms/hyperledger-fabric/configuration/roles/create/chaincode/invoke/tasks/main.yaml

```diff
@@ -8,9 +8,9 @@
 - name: "Create value file for chaincode invocation"
   include_tasks: valuefile.yaml
   vars:
-    channelcreator_query: "participants[?org_status=='new']"
```

Comment:
In case of addition of the new organization the invokation of the chaincode should be done from the new organization. The invocation of chaincode should be done by new organization while executing for the `addition of the new organization` in 1.4.x as well as v2.x and from the creator organization in case of `first network setup` in v2.x